packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -129,7 +133,7 @@ $border-style: 1px solid var(--calcite-color-border-3);
     block;
     block-size: 2px;
     content: "";
-    background-color: var(--calcite-alert-dismiss-progress-background);
+    background-color: var(--calcite-internal-alert-dismiss-progress-background);
```

Comment:
This should not be internal

Reply:
Should this one be a token that users can set?

Reply:
I think so. @SkyeSeitz is there an cross-component dismiss-progress visual pattern we should be tokenizing? I think yes.

Reply:
I added this token `--calcite-alert-dismiss-progress-background-color` and got rid of `--calcite-internal-alert-dismiss-progress-background`.